index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To review</title>
-</head>
-<body>
-    
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">TITLE</h1>
+    <p id="description">This is our project for microverse ....</p>
+    <form id="survey-form">
+
+      <label for="name" id="name-label">Enter name</label>
+      <input id="name" type="text" value="Raymond" placeholder="Enter your Name"required />
+
+      <label for="email" id="email-label">Enter email</label>
+      <input type="email" id="email" placeholder="Email"required />
+
+      <label for="number" id="number-label">Enter number</label>
+      <input id="number" type="number" min="1" max="99" placeholder="Number" />
+
+      <select id="dropdown" name="activities">
+        <option value="Occupation">What is your occupation?</option>
+        <option value="Hobbies">What do you love doing?</option>
+      </select>
```

Comment:
Don't forget to add `selected` to the default `option` that you want to preselect like this:
```
<select id="dropdown" name="activities">
        <option value="value1"> Text1 </option>
        <option value="value2" selected> Text2 </option>
</select>
```

Reply:
@anderCM 
- [ ] Well done :+1:  on making comments using code suggestions. But kindly comment in such a polite way that it doesn't seem an order. 
- For example: 
***"I think it would be better to add `selected` to the default `option` that you want to preselect like this:"***

Reply:
Thanks for feedback, I will try to be kind and polite in future comments